test/package.json

```diff
@@ -13,6 +13,7 @@
     "@typescript-eslint/parser": "4.29.0"
   },
   "dependencies": {
+    "@material-ui/styles": "^4.11.5",
```

Comment:
I think those two new dependencies are not needed. (`@material-ui/styles` and `cucumber`)

I removed them on my side and the tests are still working.